beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -825,7 +825,10 @@ impl<T: BeaconChainTypes> Worker<T> {
             | Err(e @ BlockError::InconsistentFork(_))
             | Err(e @ BlockError::ExecutionPayloadError(_))
             // TODO(merge): reconsider peer scoring for this event.
-            | Err(e @ BlockError::ParentExecutionPayloadInvalid { .. })
+            | Err(e @ BlockError::ParentExecutionPayloadInvalid { 
+                warn!(self.log, "Could not validate the payload from parent, rejecting the block";
+                "error" => %e);
+             })
```

Comment:
This syntax isn't quite right, the `| Err(e ..)` is an arm of a `match` pattern. The `{ .. }` represents the fields of the `ParentExecutionPayloadInvalid` error, _not_ the code that executes when it is matched. The code to run goes after the `=>` at the end of the pattern. As an example:

```rust
match result {
   Err(BlockError::ExecutionPayloadError(..))
   | Err(BlockError::ParentExecutionPayloadInvalid { .. }) => {
       println!("error is either execution payload error or parent execution payload invalid");
   }
   Err(BlockError::GenesisBlock) => {
       println!("error is exactly GenesisBlock");
   }
   _ => println!("something else"),
}
```

In our real code we want to single out the `ParentExecutionPayloadInvalid` and log a warning.